code/modules/clothing/outfits/plasmaman.dm

```diff
@@ -202,7 +202,7 @@
 	name = "Chief Engineer Plasmaman"
 
 	uniform = /obj/item/clothing/under/plasmaman/chief_engineer
-	gloves = /obj/item/clothing/gloves/color/plasmaman/chief_engineer
+	gloves = /obj/item/clothing/gloves/color/yellow
```

Comment:
??

Reply:
what, plasmaman needs insuls too

Reply:
engineer and chief engineer plasmaman gloves are already insulated, they just have a different sprite